submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestCleaner.kt

```diff
@@ -29,15 +36,15 @@ class SubmissionRequestCleaner(
 
         if (current != null) {
             logger.info { "${new.accNo} ${new.owner} Started cleaning common files of version ${new.version}" }
-            deleteCommonFiles(new, current)
+            runBlocking { deleteCommonFiles(new, current) }
```

Comment:
I think we should make all functions suspend it is already supported in  RabbitListener https://github.com/spring-projects/spring-amqp/pull/2460.  so we can suspend that one and all others downstream. 

If this delay PR we can block main funcitons but keep this one suspend so later is easier to suspend only the  main ones.  (Apply in others places )

Reply:
I think it's better to leave these changes for a separate PR immediately after we merge this one and the one where you made the endpoints reactive which I think will cascade up